dev-docs/howto/vpn/templates/statefulset.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "..fullname" . }}-frontend
+  labels: {{- include "..labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "..selectorLabels" . | nindent 6 }}
+      component: frontend
+  template:
+    metadata:
+      labels:
+        {{- include "..selectorLabels" . | nindent 8 }}
+        component: frontend
+    spec:
+      hostNetwork: false
+      initContainers:
+      - name: tproxy-setup
+        image: nixery.dev/busybox/iptables
```

Comment:
I think in the long run we want to be able to pin images _at least_ by version if not also by hash. We could also think about signing them. 

It should be fine to just use one image with the union of all packages needed.

We should create a follow-up ticket for this.

Reply:
Created a follow-up ticket, not going to address it in this PR though.